### PR TITLE
Added Game dev with Unity stack

### PR DESCRIPTION
**Pull request by MordodeMaru**
### For all PRs

[ ] I read the [Contribution Guide](CONTRIBUTING.md)

### For new stacks

[ ] I followed the [do's and don'ts](CONTRIBUTING.md#contributing-dos-and-donts)
[ ] This stack does not already exist

### CODE REVIEW

**Comment by MordodeMaru** [comment]
Added a game development and design simple stack

**Comment by MordodeMaru** [comment]
Added a third tool to the stack.

**Comment by jdorfman** [comment]
Hi @MordodeMaru 

Thanks so much for your contribution! We will review it and get back to you asap. =)

File: README.md [hunk 1]
```diff
@@ -248,6 +248,19 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
 - [Tutorial: Fullstack React Native with GraphQL and Authentication](https://blog.hasura.io/tutorial-fullstack-react-native-with-graphql-and-authentication-18183d13373a/)
 - [Guide: Hasura GraphQL Engine One-click App on DigitalOcean Marketplace](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html)
 
+## Unity gamedev and desing with Plastic SCM starter kit [↗](https://awesomestacks.dev/unity-gamedev-and-desing-with-plastic-scm)
```
Comment (jdorfman) [inline README.md]: Please update title to "Unity Game Development boilerplate"

File: README.md [hunk 2]
```diff
@@ -248,6 +248,19 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
...
+
+Unity is a game engine that connects to Plastic SCM to keep version control and large asset and huge repo management clean and frictionless. 
+
+- [Unity](https://unity.com/) - [🛠](https://stackshare.io/unity-3d) - [🐙](https://github.com/Unity-Technologies/UnityCsReference) - The C# part of the Unity engine and editor source code. May be used for reference purposes only.
+- [Plastic SCM](https://www.plasticscm.com/ ) - [🛠️](https://stackshare.io/plastic-scm) - [🐙](https://github.com/PlasticSCM/plastic-docker) - PlasticSCM Docker image.
+- [TeamCity](https://www.jetbrains.com/teamcity/) - [🛠](https://stackshare.io/teamcity) - [🐙] (https://github.com/JetBrains/teamcity-docker-agent) - Team City Docker image sources
```
Comment (jdorfman) [inline README.md]: Please remove  - "[🐙] (https://github.com/JetBrains/teamcity-docker-agent) - Team City Docker image sources"

File: README.md (hunk 1 above)
Comment (MordodeMaru) [reply README.md]: Will do and PR back.

File: README.md (hunk 2 above)
Comment (MordodeMaru) [reply README.md]: Will do and PR back.

**Comment by MordodeMaru** [comment]
Changes requested have been applied.

**Comment by MordodeMaru** [comment]
By mistake closed the PR. Changes requested have been applied.

**Comment by dzello** [comment]
Thanks @MordodeMaru! I've gone ahead and merged it in. I'm not sure why there was a build failure before but I'll address it if doesn't work this time.

**Pull request merged**